### ISSUE TITLE
matplotlib has no attribute colormaps when doing Sankey plots

### ISSUE THREAD

**Issue by laimaretto** [d3blocks/d3blocks#4]
Hi!

I was reading your article [here](https://towardsdatascience.com/hands-on-guide-to-create-beautiful-sankey-charts-in-d3js-with-python-8ddab43edb43) and I tried to run one of the examples:

```
from d3blocks import D3Blocks

# Initialize
d3 = D3Blocks()

# Import example
df = d3.import_example('energy')

# Link settings
d3.sankey(df, link={"color": "source-target"})
```

After doing so, I get the following error:

```
In [11]: d3.sankey(df, link={"color": "source-target"}) 
    ...:                                                                                                                                                                                                     
[d3blocks] >INFO> Create label properties based on [Set1].
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-11-c0dd4004eea9> in <module>
----> 1 d3.sankey(df, link={"color": "source-target"})

~/.local/lib/python3.8/site-packages/d3blocks/d3blocks.py in sankey(self, df, title, filepath, figsize, node, link, margin, showfig, overwrite)
    954         # Set default label properties
    955         if not hasattr(self, 'labels'):
--> 956             labels = self.get_label_properties(labels=np.unique(df[['source', 'target']].values.ravel()), cmap=self.config['cmap'])
    957             self.set_label_properties(labels)
    958 

~/.local/lib/python3.8/site-packages/d3blocks/d3blocks.py in get_label_properties(self, labels, cmap)
   1244 
   1245         # Create unique colors
-> 1246         hexcolors = colourmap.generate(len(uil), cmap=cmap, scheme='hex')
   1247         # Make dict with properties
   1248         labels = make_dict_label_properties(uil, hexcolors)

~/.local/lib/python3.8/site-packages/colourmap/colourmap.py in generate(N, cmap, method, keep_alpha, scheme, verbose)
     64     else:
     65         # base = plt.cm.get_cmap(cmap)
---> 66         base = matplotlib.colormaps[cmap]
     67         color_list = base(np.linspace(0, 1, N))[:, 0:listlen].tolist()
     68         # If there are not enough colors in the cmap, use the seaborn method.

AttributeError: module 'matplotlib' has no attribute 'colormaps'
```
I'm using python 3.8 and d3blocks 1.0.8.

Thank you!

**Comment by erdogant** [comment]
Thanks for mentioning this issue. It should be fixed now as it requires colourmap>=1.1.9.
Plz update and try again: `pip install -U d3blocks`

**Comment by laimaretto** [comment]
It's working now. Awesome work! Thank you!
